sdklab/stress_fault/tsconfig.json

```diff
@@ -1,12 +1,10 @@
 {
     "compilerOptions": {
         "outDir": "./dist",
-        "target":"es5",
+        "target":"es2017",
+        "module": "commonjs",
         "sourceMap": true,
         "declaration": true,
-        "lib": [
-            "es6"
-        ],
```

Comment:
We also have the `lib` field in `service/samples/typescript`, `service`, and `common/transport/mqtt`. Could we get rid of those too?